wikipedia_trends/wikipedia_top_viewed_genes.tsv

```diff
@@ -0,0 +1,21 @@
+gene_symbol	page_views
```

Comment:
Looks great!  Enhancement ideas (for subsequent PRs):

* Include all genes
* Add column like `page_views_change`, i.e. `page_views(today - 1)` - `page_views(today - 2)`